_posts/0000-01-02-michaelsandiegoviray.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
I think here we could have some more text regarding accessibility and clarity as what is meant to "Your text"